test/ModalSpec.js

```diff
@@ -434,6 +436,15 @@ describe('Modal', function () {
         </Modal>
         , focusableContainer);
     });
-  });
 
+    it('should not attempt to focus nonexistent children', () => {
+      const Dialog = () => null;
```

Comment:
The old logic found some weird container `<div>` (maybe from a transition or something?) as the dialog element, instead of picking up on there not being any dialog element.